Error building on macOSX 10.12.1

I was trying follow this [guide](https://github.com/deeplearning4j/libnd4j/blob/master/macOSx10%20(CPU%20only).md) to build libnd4j stack however, getting 'Undefined symbols for architecture x86_64 ' error followed by below in the end while running `buildnativeoperations.sh` script.

`ld: symbol(s) not found for architecture x86_64
collect2: error: ld returned 1 exit status
make[2]: *** [blas/libnd4j.dylib] Error 1
make[1]: *** [blas/CMakeFiles/nd4j.dir/all] Error 2
make: *** [all] Error 2
`

Error log attached at
[build.txt](https://github.com/deeplearning4j/libnd4j/files/644174/build.txt)

I primarily want to develop at dl4j java stack, however it seems above is required.

--- a/blas/CMakeLists.txt
+++ b/blas/CMakeLists.txt
@@ -125,7 +125,9 @@ elseif(CPU_BLAS)
         set (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${OpenMP_CXX_FLAGS}")
     endif()
 
-    if(APPLE OR MSYS)
+    if(APPLE)
+        SET( CMAKE_CXX_FLAGS  "${CMAKE_CXX_FLAGS} -static-libgcc -static-libstdc++")
+    elseif(MSYS)
         SET( CMAKE_CXX_FLAGS  "${CMAKE_CXX_FLAGS} -static")
     endif()
 

